easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.15.1-foss-2023b.eb

```
@@ -0,0 +1,210 @@
+    }),
+    ('astor', '0.8.1', {
+        'checksums': ['6a6effda93f4e1ce9f618779b2dd1d9d84f1e32812c23a29b3fff6fd7f63fa5e'],
+    }),
+    ('gviz-api', '1.10.0', {
+        'source_tmpl': 'gviz_api-%(version)s.tar.gz',
+        'checksums': ['846692dd8cc73224fc31b18e41589bd934e1cc05090c6576af4b4b26c2e71b90'],
+    }),
+    ('tensorboard-plugin-profile', version, {
+        'source_tmpl': 'tensorboard_plugin_profile-%(version)s.tar.gz',
+        'checksums': ['84bb33e446eb4a9c0616f669fc6a42cdd40eadd9ae1d74bf756f4f0479993273'],
+    }),
+    (name, version, {
+        'patches': [
+            'TensorFlow-2.13.0_add-missing-system-protobuf-targets.patch',
+            'TensorFlow-2.15.1_remove-libclang-dep.patch',
+            'TensorFlow-2.15.1_remove-io-gcs-filesystem-dep.patch',
+            'TensorFlow-2.15.1_add-default-shell-env.patch',
+            'TensorFlow-2.15.1_fix-flatbuffer-license.patch',
+            'TensorFlow-2.15.1_fix-pybind11-build.patch',
+            'TensorFlow-2.15.1_fix-AVX512-eigen-compilation.patch',
+            'TensorFlow-2.15.1_fix-AVX512-eigen-compilation-gcc13.patch',
+            'TensorFlow-2.15.1_upgrade-ml_dtypes-dependency-version.patch',
+        ],
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/tensorflow/tensorflow/archive/'],
+        'test_script': 'TensorFlow-2.x_mnist-test.py',
+        'test_tag_filters_cpu': '-gpu,-tpu,-no_cuda_on_cpu_tap,-no_pip,-no_oss,-oss_serial,-benchmark-test,-v1only',
+        'test_tag_filters_gpu': ('gpu,-no_gpu,-nogpu,-gpu_cupti,-no_cuda11,'
+                                 '-no_pip,-no_oss,-oss_serial,-benchmark-test,-v1only'),
+        'test_targets': [
+            '//tensorflow/core/...',
+            '-//tensorflow/core:example_java_proto',
+            '-//tensorflow/core/example:example_protos_closure',
+            '//tensorflow/cc/...',
+            '//tensorflow/c/...',
+            '//tensorflow/python/...',
+            '-//tensorflow/c/eager:c_api_test_gpu',
+            '-//tensorflow/c/eager:c_api_distributed_test',
+            '-//tensorflow/c/eager:c_api_distributed_test_gpu',
+            '-//tensorflow/c/eager:c_api_cluster_test_gpu',
+            '-//tensorflow/c/eager:c_api_remote_function_test_gpu',
+            '-//tensorflow/c/eager:c_api_remote_test_gpu',
+            '-//tensorflow/core/common_runtime:collective_param_resolver_local_test',
+            '-//tensorflow/core/kernels/mkl:mkl_fused_ops_test',
+            '-//tensorflow/core/kernels/mkl:mkl_fused_batch_norm_op_test',
+            '-//tensorflow/core/ir/importexport/tests/roundtrip/...',
+        ],
+        'testopts': "--test_env=HOME=/tmp --test_timeout=3600 --test_size_filters=small ",
+        'testopts_gpu': ('--test_env=HOME=/tmp --test_timeout=3600 --test_size_filters=small '
+                        '--run_under=//tensorflow/tools/ci_build/gpu_build:parallel_gpu_execute '),
```

```suggestion
                         '--run_under=//tensorflow/tools/ci_build/gpu_build:parallel_gpu_execute '),
```
Apologies - one extra space needed for alignment.